bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -16,6 +16,8 @@
 
 package com.google.cloud.bigtable.grpc;
 
+import static com.google.cloud.bigtable.util.ThreadPoolUtil.createThreadFactory;
```

Comment:
Why not import ThreadPoolUtil

and then use ThreadPoolUtil.createThreadFactory?